services/util/src/main/java/tech/pegasys/pantheon/services/util/RocksDbUtil.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.services.util;
+
+import tech.pegasys.pantheon.util.InvalidConfigurationException;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.rocksdb.RocksDB;
+
+public class RocksDbUtil {
```

Comment:
I created this helper package for common code within services (which is for now just this rocksdb init code).  WDYT @ajsutton ?

Reply:
Yep that works for me.